algorithms/CPlusPlus/Stacks/reverse-stack.cpp

```diff
@@ -0,0 +1,60 @@
+#include <iostream>
```

Comment:
add a user friendly description about the program

Reply:
done